flexget/plugins/list/series_list.py

```diff
@@ -0,0 +1,438 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+from collections import MutableSet
+from datetime import datetime
+
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+from sqlalchemy import Column, Unicode, Integer, ForeignKey, Boolean, DateTime, String, func
+from sqlalchemy.orm import relationship
+from sqlalchemy.orm.collections import InstrumentedList
+from sqlalchemy.sql.elements import and_
+
+from flexget.plugins.filter.series import FilterSeriesBase
+from flexget import plugin
+from flexget.db_schema import versioned_base
+from flexget.utils.database import json_synonym, with_session
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.tools import split_title_year
+
+log = logging.getLogger('series_list')
+Base = versioned_base('series_list', 0)
+
+SUPPORTED_IDS = FilterSeriesBase().supported_ids
+SETTINGS_SCHEMA = FilterSeriesBase().settings_schema
+SERIES_ATTRIBUTES = SETTINGS_SCHEMA['properties']
+
+
+class SeriesListList(Base):
```

Comment:
i retained the naming convention we used by other plugins. `PluginName` `TableType`